### ISSUE TITLE
Add ability to run JS script with no manifest

### ISSUE THREAD

**Issue by fatcerberus** [spheredev/neosphere#47]
Implementing this would allow minisphere to run a bare JS script with the full set of Sphere bindings without the need for an `.sgm` or `.s2gm` file.

Since there would be no manifest to anchor the sandbox, any paths referenced by such a script would have to be treated as relative to the directory containing it.
- [x] Implement ability to initilaize a sandbox with only a script
- [x] Figure out what to do about `GetGameManifest()`
- [x] Improve resolution handling to support operation without a manifest - Contingent on #48

**Comment by fatcerberus** [comment]
There are a few things that would break without a manifest: 
- GetGameManifest(), obviously. No metadata would be available to the script because none would exist.
- Because there is no metadata, what to display in the title bar is not clear, probably just the script filename.
- Screen resolution.  Currently this is defined by the manifest, but with a bare script there would need to be an alternate way to set it.  This almost _requires_ implementing an API to change the resolution at runtime...
- And maybe a few other things I'm forgetting...

**Comment by fatcerberus** [comment]
For `GetGameManifest()`, it might make sense to return a dummy manifest containing only a `script` property--which would point to the absolute path of the script used to initialize the sandbox.

Alternatively, it could just return null or undefined, which might make more sense semantically, and would allow easy detection of bare scripts at runtime.

**Comment by fatcerberus** [comment]
This feature has been implemented as of a4879e39c41b7f7b122db470e7dd193b8f874dc6.

Given a JS script containing a `game()` function, you may enter on the command line:

```
msphere script.js
```

And minisphere will run the script.  A game manifest will be synthesized, and the directory containing the script being executed will be used as the SphereFS root.  The synthesized manifest looks like this in JSON:

``` json
{
    "name": "script.js",
    "author": "Author Unknown",
    "summary": "/path/to/script.js",
    "resolution": "320x240",
    "script": "script.js"
}
```

This is what will be returned if the script calls `GetGameManifest()`, as well as internally when engine code calls `get_game_manifest()`.  Thus bare-script operation is completely transparent outside of SphereFS file system code.
